blog/2020-07-31-temporal-v0.28.0-changelog.md

```diff
@@ -0,0 +1,113 @@
+---
+tags:
+- changelog
+- temporal
+- v1
+posted_on_: 2020-07-31T00:00:00
+id: temporal-v0.28.0-changelog
+title: Temporal v0.28.0 changelog
+author: Cully Wakelin
+author_title: SDE Technical Writer
+author_image_url: https://avatars2.githubusercontent.com/u/34380806?s=400&u=5cd38b5e4416a5d10cdf9ebd386eec1d02f0b067&v=4
+release_version: v0.28.0
+---
+
+<!--truncate-->
+
+Dear Temporal users,
+
+Today is an exciting day for us as Temporal is releasing its newest version, v0.28.0, which  brings us closer to a stable production ready release. Thank you to all of the users who have helped us iterate and move the product forward.
```

Comment:
"as Temporal is releasing", say this from a position of beloning. Right now it sounds like you're not part of the company.

"which  brings us clos"

an extra space between which.

also I would reword to say

"which is our first major step towards a production ready release. 

"Thank you to all of"

I want to thank all of the users

Reply:
> "as Temporal is releasing", say this from a position of beloning. Right now it sounds like you're not part of the company.

"Today is an exciting day for us, as Temporal is releasing..." ?? "us" is used in the first clause of the sentence.

> "which brings us clos"
 Fixed.

> also I would reword to say
> "which is our first major step towards a production ready release.

Its not - its ONE more step closer - it could be argued that any of the previous releases were the FIRST major step... 

> "Thank you to all of"
> I want to thank all of the users - 

These are not my changes. I am drafting and publishing this on behalf of the company and the engineers. So, I prefer to keep this generally inclusive "We thank all of the users". I will be personal in chats, and discourse posts, or maybe specific blog posts. This doesn't seem like the right scenario to show this type of ownership.